docs/src/content/docs/guides/onetime-remittance-fixed-receive.mdx

```diff
@@ -0,0 +1,512 @@
+---
+title: Send a remittance with a fixed receive amount
+---
+
+import { LinkOut } from '@interledger/docs-design-system'
+import { Tabs, TabItem, Badge } from '@astrojs/starlight/components'
+import Start from '/src/partials/grant-start-interaction.mdx'
+import Finish from '/src/partials/grant-finish-interaction.mdx'
+
+:::tip[Summary]
+Learn how to send a one-time remittance payment where the recipient will receive a fixed amount.
+:::
+
+A remittance payment is a transfer of money from one person to another, typically across borders or long distances, often involving currency conversion. In this guide, you will learn how to implement a one-time remittance payment feature where your user can specify exactly how much the recipient should receive.
+
+This approach is particularly useful for remittance app scenarios where:
+
+- The sender and the recipient each transact in different currencies
+- The sender wants the recipient to receive a fixed amount denominated in the recipient's local currency
+- The sender is willing to cover any differences in the exchange rate
+
+## Scenario
+
+Imagine someone in the US wants to send money to a family member in Mexico. They want their family member to receive _exactly_ $5,000 Mexican pesos (MXN), even if it means paying a slightly higher amount in US dollars to cover currency conversion. This is different from a payment where the sender specifies exactly how much to send, regardless of what the recipient will receive.
```

Comment:
> even if it means paying a slightly higher amount in US dollars to cover currency conversion

Maybe we can say, "...irrespective/regardless of what the currency conversion will be"? They'll always have to cover the currency conversion.